indico/web/client/js/react/components/TooltipIfTruncated.jsx

```diff
@@ -11,15 +11,21 @@ import PropTypes from 'prop-types';
 export default class TooltipIfTruncated extends React.Component {
   static propTypes = {
     children: PropTypes.any.isRequired,
+    customText: PropTypes.string,
```

Comment:
`customText` vs `text` vs `tooltip`.. any opinions @indico/development-team

Reply:
Hm.. `tooltip` seems like a good name for that (I would be more in favor of `customText` if `element.innerText` had precedence over the prop value, but apparently it is not the case)

Reply:
I think I'll just go with `tooltip`. What do you think?